### ISSUE TITLE
Gantt zoom

### ISSUE THREAD

**Issue by toni-sharpe** [toni-sharpe/brainstem-research#28]
### Problem

The `Gantt` would be better if it would zoom

### Solution

To allow the chart to toggle between a full screen for the whole thing, to a full screen for a single division with left and right arrow keyboard scrolling, and buttons, 

The zoom should be accompanied by a tiny version that shows which segment is being zoomed into
